integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -61,20 +63,26 @@ import com.okta.sdk.resource.application.SwaApplicationSettingsApplication
 import com.okta.sdk.resource.application.WsFederationApplication
 import com.okta.sdk.resource.application.WsFederationApplicationSettings
 import com.okta.sdk.resource.application.WsFederationApplicationSettingsApplication
+import com.okta.sdk.resource.apps.CSRMetadata
```

Comment:
the tag for this is `apps` and above is `application` this should be consistant

Reply:
Agreed. Will update the spec.